includes/auth/class-auth-init.php

```
@@ -0,0 +1,303 @@
+            if (file_exists($file)) {
+                require_once $file;
+            }
+        });
+    }
+    
+    /**
+     * Initialize auth manager
+     */
+    private static function init_auth_manager() {
+        // Get instance to trigger initialization
+        $auth = ZP_Auth_Manager::get_instance();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$auth'. (Unused Code Rules)

(UnusedLocalVariable)